os/arch/arm/src/armv7-m/mpu.h

```diff
@@ -660,6 +660,16 @@ static inline void mpu_peripheral(uint32_t region, uintptr_t base, size_t size)
 	putreg32(regval, MPU_RASR);
 }
 
+static inline void up_set_mpu_app_configuration(struct tcb_s *rtcb) {
```

Comment:
Put `{` at next line. coding rule

Reply:
How about below?
```
#if defined(MPU) && defined(SEPARATION)
static inline void up_xxx_configuration(struct tcb_s *rtcb)
{
  ...
}
#else 
#define up_xxx_configuration(x)
#endif 
```
Then, when it is not available, we can avoid meanless function-call.

Reply:
Done.